learn/why-ballerina/flexibly-typed.md

```diff
@@ -67,9 +67,9 @@ A type in Ballerina represents a set of possible shapes it can have. So any valu
 
 ### Subtypes in Ballerina
 
-Subtyping in Ballerina is semantic. It is defined through shapes, where `S` is a subtype of `T` if the set of shapes denoted by `S` are a subset of the shapes denoted by `T`. The examples below demonstrate this behavior.
+Subtyping in Ballerina is semantic. It is defined through shapes, where `S` is a subtype of `T` if the set of shapes denoted by `S` is a subset of the shapes denoted by `T`. The examples below demonstrate this behavior.
 
-The type `boolean` is a simple basic type in Ballerina without a storage identity, so its values become equivalent to its shapes. Therefore, the `boolean` type is defined as having two shapes, `true` and `false`.
+The type `boolean` is a simple basic type in Ballerina without storage identity, so its values become equivalent to its shapes. Therefore, the `boolean` type is defined as having two shapes, `true` and `false`.
```

Comment:
```suggestion
The type `boolean` is a simple basic type in Ballerina without storage identity so that its values become equivalent to its shapes. Therefore, the `boolean` type is defined as having two shapes, `true` and `false`.
```

Reply:
+1 Then. let's just remove the comma after `so` as it is redundant.

Reply:
```suggestion
The type `boolean` is a simple basic type in Ballerina without storage identity so its values become equivalent to its shapes. Therefore, the `boolean` type is defined as having two shapes, `true` and `false`.
```

Reply:
Fixed.